### ISSUE TITLE
Character Level Snapshot

### ISSUE THREAD

**Issue by nightwing367** [aurorabuilder/elements#140]
TLDR : Please add a feature that allows a high level character to generate a low level preview character sheet.

I'm one of the guys that likes to plan ahead his characters. At the moment, if I make a lvl 20 build and wish to start playing that character at level 1, I need to make a second character save and copy manually the lvl ups from the lvl 20 save or remove levels 1 by one from the character which is very slow (performance & double validation) and if I save for some reason, will make me loose my whole build. This is pretty much the only thing that annoys me in this great app at the moment so a fix would be very appreciated.

If that functionality is already present, I haven't found it. Don't hesitate to ask for more information.
Thanks in advance, you guys rock!

**Comment by NickVendel** [comment]
...is it hard to just put your progression into Textpad file?
I'm guessing it's some crazy-amazing build with multiclassing and stuff, but it's just 20 or so steps, i don't think it's that hard to do yourself.
Or maybe i misunderstood question.

**Comment by swdriessen** [comment]
I can look into this, no guarantees so for now you'll have to keep two character files if you want to do this.

**Comment by nightwing367** [comment]
@NickVendel For D&D 3.5th I used to do that a lot. There is a lot of things to keep in mind :
class
class features (for overview and character progression flow)
class choices (like specialization, warlock invocations)
spells
feats/ability increase.
magical item wishlist
etc...

It can definitely be done by text file but this app can do it very well. I'd be an idiot not to request it. If it doesn't get done well that's too bad. I can live with that.

@swdriessen I'd like to mention that I'm willing to offer some assistance if you need help with the coding (any of it, not just this feature). I happen to work as a software architect mostly in QA. A pet project might help me not get rusty.

**Comment by NickVendel** [comment]
@nightwing367 I don't know how you imagine implementation of something like this. I picture only that Aurora would create text file with all the choices you made level by level... but that's kinda it.
Anyway, i understand sometimes this can be hard to do, but in 5E it's a lot easier than in 3.5, since there's less requirements for pretty much anything (no more "you need this amount of points in these skills to buy this Feat" or something).

**Comment by swdriessen** [comment]
@nightwing367 Thanks, but for now it will remain my solo hobby project. It's first 'big' project and i want to see it through to a proper v1.0 and get rid of that *preview* tag. I have some plans for the future where some collaboration will be possible to improve the builder.

Also, i don't want to be responsible for your heart attack from looking at my code as a QA 😄

**Comment by nightwing367** [comment]
I see that I might not have been clear enough. I'm interested in the "previews" so I can print/pdf my character sheets at X level so I can make my character at lvl 20 (or just higher level that X) up front and then simply print/pdf a new character sheet whenever I level without having to fiddle with the choices every level if that makes sense. It doesn't have to cascade through the whole sheet, just the end result. English isn't my first language so sometimes my meaning gets twisted a bit between what my mind thinks and what my fingers type. =)
